website/package.json

```diff
@@ -14,33 +14,23 @@
     "check:spelling": "./node_modules/.bin/spellchecker --quiet --files=\"docs/**/*.{md,mdx}\" --dictionaries=\"./dictionary.js\" --reports=\"./spelling.json\" --plugins spell indefinite-article repeated-words syntax-mentions syntax-urls frontmatter"
   },
   "engines": {
-    "node": ">=14.15.0",
+    "node": ">=16.13.1",
     "npm": ">=7"
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
+  "browserslist": [
+    "defaults"
+  ],
   "dependencies": {
     "@docusaurus/core": "^2.0.0-beta.0",
     "@docusaurus/module-type-aliases": "^2.0.0-beta.0",
     "@docusaurus/plugin-google-gtag": "^2.0.0-beta.0",
     "@docusaurus/preset-classic": "^2.0.0-beta.0",
-    "@types/classnames": "^2.3.1",
```

Comment:
Removed, since it is not needed anymore https://www.npmjs.com/package/@types/classnames